CHANGES.rst

```diff
@@ -1,6 +1,21 @@
 Changes
 =======
 
+Version master (unreleased)
+---------------------------
+
+- Adds new utility to send emails.
+- Adds centralised operational options validation.
+- Fixes `memory link in Bravado client instantiation <https://github.com/reanahub/reana-server/issues/225>`_.
```

Comment:
If we are going to link to GitHub issues in release notes, then let's do it uniformly everywhere, e.g. the following style:

```
- Adds new utility to send emails. (#124)
- Fixes memory link in Bravado client instantiation. (#225)
- ...
```

It's a style used in several projects including K8s.